pages/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994.md

```diff
@@ -1,14 +0,0 @@
----
-title: Apply for education benefits
-entryname: 0994-edu-benefits
-layout: page-react.html
-description: Vet Tec.
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/education/">Education</a></li>
-    <li><a aria-current="page" href="/education/how-to-apply/">Apply for VET TEC</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-a7d109a9115abd7fe1ddbeae871b3bddR5. Breadcrumbs look okay.